Sources/IdentityConstants.swift

```diff
@@ -24,9 +24,22 @@ enum IdentityConstants {
         static let ZERO_ADVERTISING_ID = "00000000-0000-0000-0000-000000000000"
     }
 
-    enum SharedStateKeys {
-        static let CONFIGURATION = "com.adobe.module.configuration"
-        static let IDENTITY_DIRECT = "com.adobe.module.identity"
+    enum SharedState {
```

Comment:
Note, I moved some constants around so they are similar to Edge. There will be several line diff just for that change.